modules/ROOT/pages/faq.adoc

```diff
@@ -193,3 +193,20 @@ Kubernetes cluster, you will be able to use upcoming tools to coordinate
 updates and reboots.
 
 https://discussion.fedoraproject.org/t/launch-faq-how-do-i-coordinate-cluster-wide-os-updates-is-locksmith-or-the-container-linux-update-operator-available-for-fedora-coreos/56[discuss]
+
+== How do I upload Fedora CoreOS to private AWS EC2 regions?
+
+Fedora CoreOS today is only uploaded to the public AWS regions. For private
+regions like GovCloud and AWS China, you must upload the images yourself.
+
+Note that Fedora CoreOS uses a unified BIOS/UEFI partition layout. As such, it
+is not compatible with the default `aws ec2 import-image` API (for more
```

Comment:
How is it the default?  Can we drop the word `default`?

Reply:
It's the "default" insofar as it's what I understand a more common workflow than `import-snapshot` for people who want to create AMIs from local disk images. But not against dropping the word entirely. :)